src/SFA.DAS.AssessorService.Application.UnitTests/Handlers/RegisterUpdate/WhenRegisterUpdateCalled_AndItReturnsUnknownEpao.cs

```diff
@@ -26,13 +26,13 @@ public void Arrange()
                     new OrganisationSummary {Id = "EPA0003", Ukprn = 33333333}
                 }.AsEnumerable()));
 
-            ApiClient.Setup(c => c.Get("EPA0003")).Returns(new ExternalApis.AssessmentOrgs.Types.Organisation { Id = "EPA0003", Name = "A New EPAO", UkPrn = 33333333});
+            ApiClient.Setup(c => c.Get("EPA0003")).Returns(new Organisation { Id = "EPA0003", Name = "A New EPAO"}); 
```

Comment:
Don;t know if it helps but I am using  <PackageReference Include="RichardSzalay.MockHttp" Version="4.0.0" />

there are examples in SFA.DAS.AssessorService.Web.Staff.Tests.MockedObjects:public class MockedApiClient

might be worthwhile  a look.